app.py

```
@@ -1,91 +1,128 @@
-    """
-    st.markdown(disclaimer, unsafe_allow_html=True)
+    """, unsafe_allow_html=True)
 
 
 def main():
     """Streamlit entry point for StoDir."""
     st.title("StoDir - Stock Direction Forecasting")
 
+    config, model = load_config_and_model()
+    # Using an assertion to ensure the model and config are loaded before proceeding.
+    assert model is not None and config is not None, "Model or config failed to load. Cannot proceed."
```

Ruff (0.12.2)

Use of `assert` detected

(S101)